albinos-editor/src/albinospkg/cli.nim

```diff
@@ -2,24 +2,55 @@ import terminal
 import strutils
 import rdstdin
 import linenoise
+import cmd
+import ../libalbinos/albinos
 
-let doc = """
-Usage:
-    help (clear the screen)
-    exit (quit the cli)
-    clear (clear the terminal)
-    config create <name> (create a config with the given name)
-    config load <key> (load a config with the given key)
+const help_cmd_msg = "\t\e[93mhelp\e[39m (show this message)\n\t"
```

Comment:
Don't colour strings change depending on the terminal ? Also you probably need a way to turn them off for terminals that don't support them.

Reply:
It's supported even in TTY mode at least on xterm.

Reply:
https://misc.flogisoft.com/bash/tip_colors_and_formatting